build/Build.Steps.cs

```diff
@@ -333,22 +335,21 @@ partial class Build
                 return;
             }
 
-            IEnumerable<TargetFramework> frameworks = IsWin ? TestFrameworks : TestFrameworks.ExceptNetFramework();
+            var frameworks = IsWin ? TestFrameworks : TestFrameworks.ExceptNetFramework();
 
             for (int i = 0; i < TestCount; i++)
             {
-                DotNetTest(config => config
+                DotNetMSBuild(config => config
                     .SetConfiguration(BuildConfiguration)
-                    .SetTargetPlatform(Platform)
+                    .SetPlatform(Platform)
                     .SetFilter(AndFilter(TestNameFilter(), ContainersFilter()))
                     .SetBlameHangTimeout("5m")
                     .EnableTrxLogOutput(GetResultsDirectory(project))
-                    .SetProjectFile(project)
-                    .EnableNoRestore()
-                    .EnableNoBuild()
-                    .CombineWith(frameworks, (s, fx) => s
-                        .SetFramework(fx)
-                    ));
+                    .SetTargetPath(project)
+                    .DisableRestore()
+                    .SetTargets("VSTest")
```

Comment:
Can we move these (L350-351) also to the extensions. 
I guess the extension method name will be more descriptive than the lines alone.